packages/site/pages/components/button.js

```diff
@@ -1,5 +1,5 @@
 import core from '@pluralsight/ps-design-system-core'
-import Button from '@pluralsight/ps-design-system-button/react.js'
+import Button from '@pluralsight/ps-design-system-button'
```

Comment:
This is a good usage to replace.  Seems like maybe we want to replace (all) others?  ...Make this the default usage.  Where else might that be?  Storybooks, probably...

Reply:
Cool I'll add that. Wasn't sure how a big a change we wanted to do on what's a prototype export pattern right now.